data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1298 @@ mission "Unfettered Jump Drive Trading"
+			`	You ask around for Mera, the Unfettered engineer you met before. She said she would tell you more about it once she gets the Geocoris logs, but you have yet to see her since you brought those logs to the local authorities. Hopefully, she can be trusted. After you spend an hour talking with the local Unfettered and drinking, she finally arrives. She seems to be in quite a hurry.`
+			`	"Sorry I could not see you earlier; I was busy. I know I have promised you information, but there is a more pressing matter we should focus on. A human ship will enter <system> in a few days. I need you to scan it for me with the same scanner we gave you, and then I will meet you back here with the others and answer any questions you may have," she promises.`
+			`	"Oh, and before I forget, you should not attack them, even if they strike first. I am not sure how they will even react to your presence," she adds.`
+			choice
+				`	"Okay, I better get going right now then."`
+					accept
+				`	"Why not just do it yourself?"`
+			`	"We don't know what their reaction could be. It would be preferable if you did it because they are more likely to think of you as separate from us. We value trust a lot, with anyone. Once it is damaged, even by an misinterpreted act as simple as this, it cannot easily be repaired." She still seems to be in a hurry, but you figure one more minute would not change much in your ability to catch that ship.`
+			choice
+				`	"Seems good to me. I'll do it."`
+					accept
+				`	"What kind of ship are we talking about?"`
+			`	She seems anxious to tell you. "It's a human ship, and quite a big one. We do not know its designation in human space, but it carries a few small ships and uses alien technologies. You do not need to be able to destroy it; just scan it and then come back here." She did not seem to want to answer the last question, and maybe only answered this one so you would go away. Now she is waiting for an answer. Maybe you should reassure her in your abilities before she starts doubting you and hiring someone else.`
+			choice
+				`	"I'm not worried about any one ship. I'll do it."`
+					accept
+				`	"That seems too dangerous to me. Sorry, but I will not be helping you anymore."`
+					decline
+	waypoint "Hi Yahr"
+	on waypoint
+		dialog `As you enter the system, you see the ship and identify it as a Navy Auxiliary. Whatever happens, the Unfettered asked you not to destroy it and only scan it - at least for now.`
+	npc save "scan cargo"
+		government "Alpha"
+		personality entering timid staying target
+		system "Hi Yahr"
+		ship "Alpha Auxiliary (Bringing Supplies)" "Settler"
+		conversation
+			`You have finished scanning the Auxiliary. There can be no doubt: it is controlled by Alphas. As you prepare yourself to exit the system and bring the logs back, you receive a message from the local Unfettered authorities.`
```

Comment:
Why can there be no doubt? To my knowledge the player has no way of determining that at a distance at all.

Reply:
It is painted pink, also you just scanned it using the unfettered scanner that marks life forms, presumably